addons/mail/static/src/model/model_manager.js

```diff
@@ -34,19 +34,29 @@ class ModelManager {
          * The messaging env.
          */
         this.env = env;
+
+        //----------------------------------------------------------------------
+        // Various variables that are necessary to handle an update cycle. The
+        // goal of having an update cycle is to delay the execution of computes,
+        // life-cycle hooks and potential UI re-renders until the last possible
+        // moment, for performance reasons.
+        //----------------------------------------------------------------------
+
+        /**
+         * Set of records that have been created during the current update
+         * cycle. Useful to trigger _created.
```

Comment:
_"[...] `_created()` hook methods."_

- "\`" highlights name of something in code;
- brackets highlight it's refers to a method;
- "hook methods" clarifies why it makes sense to call these methods. Plural in methods also implicitly refers to update cycle that usually call several such hook methods, so it really helps warming up reader by outlining a few bits of the logic in update cycle.